packages/notebook/src/panel.ts

```diff
@@ -75,6 +80,25 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     });
   }
 
+  _onSave(sender: DocumentRegistry.Context, state: string) {
+    if (state === 'completed') {
+      // Find markdown cells
+      const { cells } = this.model;
+      each(cells, (cell: ICellModel) => {
+        if (cell.type === 'markdown') {
+          const markdowncell: MarkdownCellModel = cell as MarkdownCellModel;
+          const attachments: ReadonlyArray<string> =
+            markdowncell.attachments.keys;
+          for (let i = 0; i < attachments.length; i++) {
+            if (!markdowncell.value.text.includes(attachments[i])) {
```

Comment:
This is not very robust (e.g. compared to the notebook implementation), but I'd be happy to merge with this as-is, and then create a follow-up issue.